mappings/net/minecraft/client/render/entity/LivingEntityRenderer.mapping

```diff
@@ -3,15 +3,23 @@ CLASS net/minecraft/class_922 net/minecraft/client/render/entity/LivingEntityRen
 	FIELD field_4737 model Lnet/minecraft/class_583;
 	FIELD field_4738 features Ljava/util/List;
 	METHOD method_18656 getYaw (Lnet/minecraft/class_2350;)F
+	METHOD method_23185 getWhiteOverlayProgress (Lnet/minecraft/class_1309;F)F
+		ARG 1 entity
+		ARG 2 tickDelta
+	METHOD method_23622 getOverlay (Lnet/minecraft/class_1309;F)I
+		ARG 0 entity
+		ARG 1 whiteOverlayProgress
 	METHOD method_4039 getLyingAngle (Lnet/minecraft/class_1309;)F
 	METHOD method_4042 scale (Lnet/minecraft/class_1309;Lnet/minecraft/class_4587;F)V
 		ARG 1 entity
 	METHOD method_4044 getHandSwingProgress (Lnet/minecraft/class_1309;F)F
 		ARG 1 entity
 		ARG 2 tickDelta
-	METHOD method_4045 getAge (Lnet/minecraft/class_1309;F)F
+	METHOD method_4045 getCustomAngle (Lnet/minecraft/class_1309;F)F
```

Comment:
What angle is this?

Reply:
It's used for a custom animation that depends on the entity. Most entities don't use it, but
 - For the wolf, it makes the tail move up and down
 - For the chicken, it makes it bob left and right
 - For the squid, it rotates slightly in the water